compose/control-plane/config.json

```diff
@@ -18,6 +21,96 @@
         "metric_system_name": "hits",
         "delta": 1
       }
-    ]
+    ],
+    "auth_config": {
+      "path": "static/threescale_wasm_auth.wasm",
+      "wasm_config": {
+        "system": {
+          "cluster_name": "a_cluster",
+          "url": "https://a-system-url/",
```

Comment:
does this work with Docker-compose?

Reply:
docker-compose does not do anything with this file, it's the config file for the control plane, and this entry is the config file for the wasm module, so not important for the CP either, we just look for the `backend` entry to set up a cluster, but even that is just a small hack, since the proper thing to do would be to have a wasm interface that declares any clusters it needs based on the configuration passed in as a raw string

Reply:
but 3scale-saas-backend should be declared as a cluster, no?

Reply:
Yes, it is [done here](https://github.com/3scale/gateway-ng-controller/pull/31/commits/1ca3db819af8ffff4e4c0f100ac1ea2c1e6758d1#diff-5c6f87176b7f64fbbeabbc31f5ad377edc268ebeb231eb55ee31c14ec3834f23R107-R116), the authrep module contains the minimal parsing via serde to extract the needed cluster from the backend section of the configuration.